source/PlayerInfo.cpp

```diff
@@ -1773,35 +1773,38 @@ bool PlayerInfo::TakeOff(UI *ui, const bool distributeCargo)
 	for(const Mission *mission : missionsToRemove)
 		RemoveMission(Mission::ABORT, *mission, ui);
 
-	// Any ordinary cargo left behind can be sold.
+	// Any ordinary cargo left behind can be either sold, stored or dumped.
```

Comment:
The thing is, if I'm not mistaken, `leftOver` also counts outfits. 
Maybe this would be better : `// Any ordinary cargo/outfits left behind should be either sold, stored or dumped.` ?